app/src/proxy.ts

```diff
@@ -0,0 +1,71 @@
+import child_process from "node:child_process";
+
+export type ProxyRequest = {
+  method: string;
```

Comment:
I think we can tighten this up a bit with:

```suggestion
  method: "GET" | "POST" | "DELETE";
```